### ISSUE TITLE
Problems installing in NS 3.x

### ISSUE THREAD

**Issue by ralph-fuechtenkort** [pubkey/rxdb#226]
## Case
BUG
## Issue
I try to install RXDB as described in a blank Nativescript 3.x --ng applicaton.
AS soon i use:
var RxDB = require('rxdb'); 
i receive an error that module crypto is missing.

## Info
- Environment: (Node.js/browser/electron/etc.)
nativescript, android on device. Same when used on IOS simulator,
- Adapter: (IndexedDB/Localstorage/LevelDB/etc.)
n/a

## Code

```javascript
import { Injectable } from "@angular/core";
require('babel-polyfill'); //only needed when you dont have polyfills 
var RxDB = require('rxdb');
import { Item } from "./item";
```

**Comment by pubkey** [comment]
Hi @ralph-fuechtenkort 
I just want to let you know that I have this issue in mind and a solution will come in the near future.
My steps are now to implement custom-builds (like in pouchdb) so we can try to use rxdb in nativescript without the crypto-module. Maybe we than can use an alternative crypto-module that works with nativescript.
I'm sorry for not creating a working nativescript-example before I claimed that rxdb works with nativescript in the readme.

**Comment by pubkey** [comment]
NOTE: Custom builds comming [soon](https://github.com/pubkey/rxdb/tree/cutsom-builds)

**Comment by ralph-fuechtenkort** [comment]
Hi sorry for the late reply, but i was on a trip last week.
That sounds great because we are considering moving from our ionic/meteor to a nativescript pouch/couch solution. Your db would have a main role in that :)
Just let me know when i can try again.
Thanks in advance and greets from Manila Ralph

**Comment by pubkey** [comment]
Hi @ralph-fuechtenkort 
I released 5.3.0 a few minutes ago. You now can do a [custom-build](https://pubkey.github.io/rxdb/CustomBuilds.html) without the crypto-module.

Example-code:

```javascript

/**
 * custom build
 */
import RxDB from 'rxdb/dist/es/core';

// import modules

import RxDBSchemaCheckModule from 'rxdb/dist/es/modules/schema-check';
RxDB.plugin(RxDBSchemaCheckModule); // this should be used in dev-mode only

import RxDBValidateModule from 'rxdb/dist/es/modules/validate';
RxDB.plugin(RxDBValidateModule);
import RxDBLeaderElectionModule from 'rxdb/dist/es/modules/leader-election';
RxDB.plugin(RxDBLeaderElectionModule);

```

Please try this out.

**Comment by pubkey** [comment]
Closing this for now. The proposed solution should work. Comment here if it does not.
